homeassistant/components/vacuum/__init__.py

```diff
@@ -77,19 +77,19 @@
 class VacuumEntityFeature(IntFlag):
     """Supported features of the vacuum entity."""
 
-    TURN_ON = 1
-    TURN_OFF = 2
+    TURN_ON = 1  # Deprecated, not supported by StateVacuumEntity
```

Comment:
why not remove it to avoid confusion?

there are only 3 lefts which are not StateVacuumEntity) if i count right:

StateVacuumEntity
 demo
 litterrobot
 mqtt
 neato
 roborock
 romy
 roomba
 sharkiq
 template
 tuya
 xiaommmmi_miio
 
VacuumEntity (wich is deprecated)
 demo
 mqtt
 ecovacs

cause demo does not count and mqtt is generic, only ecovacs will be hard to test after porting?
at least i would mark the demo and the class VacuumEntity

personal i would prefer removing VacuumEntity completly don't know side effects/effort here but it would help a lot for new vacuum implementation that they don't get confused